sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java

```diff
@@ -693,7 +698,9 @@ private FacilityReferenceDto getLabReference(LabMessageDto labMessageDto) {
 	}
 
 	private void createPathogenTest(SampleDto sampleDto, LabMessageDto labMessageDto) {
-		PathogenTestDto pathogenTestDto = buildPathogenTest(sampleDto, labMessageDto);
+		// TODO currently just the first testReport is picked here. That must be temporary.
```

Comment:
Is this gonna be fixed in another ticket?

Reply:
Yes, this part was moved to #5899.